src/Store.spec.lua

```diff
@@ -217,7 +217,7 @@ return function()
 
 			expect(caughtState.Value).to.equal(2)
 			expect(caughtAction.type).to.equal("ThrowError")
-			expect(caughtErrorResult.message).to.equal("Caught error in reducer")
+			expect(string.find(caughtErrorResult.message, "Caught error in reducer ")).to.be.ok()
```

Comment:
Doesn't look like this test is actually checking that the function name is present?